### PR TITLE
Commercial components mobile fixes

### PR DESCRIPTION
**Pull request by zeftilldeath**
Before/after
screen shot 2015-10-15 at 17 28 59

Before/after
screen shot 2015-10-15 at 17 33 56

### CODE REVIEW

File: static/src/stylesheets/components/pasteup-buttons/src/_buttons.scss [hunk 1]
```diff
@@ -223,6 +223,9 @@ Show more button:
     @include fs-textSans(1);
     line-height: $gs-baseline * 2 - 2px;
     font-weight: bold;
+    white-space: nowrap;
```
Comment (jbreckmckye) [inline static/src/stylesheets/components/pasteup-buttons/src/_buttons.scss]: Would we rather buttons be truncated, or fall to a two-line design?

Comment (zeftilldeath) [inline static/src/stylesheets/components/pasteup-buttons/src/_buttons.scss]: From what I've seen other buttons on the site truncate. A two line design looks pretty bad (especially when choosing where to place the arrow). I think as the only thing lost at 320px is the arrow this doesn't compromise usability. This is the longest example of a small cta in a commercial component I've managed to find.

Comment (jbreckmckye) [inline static/src/stylesheets/components/pasteup-buttons/src/_buttons.scss]: If that's what the other buttons do, then we should act consistently, yeah.

**Comment by jbreckmckye** [comment]
All looks good to me! :+1:

**Pull request merged**